src/auth.ts

```diff
@@ -0,0 +1,6 @@
+import NextAuth from "next-auth"
+import Google from "next-auth/providers/google"
+ 
+export const { handlers, auth, signIn, signOut } = NextAuth({
+  providers: [Google],
```

Comment:
Let's update the `env.sample` with the environment variables and update the README for the auth related stuff?